dynamic_paperclip.gemspec

```diff
@@ -20,12 +20,12 @@ Gem::Specification.new do |s|
   s.add_dependency "paperclip", ">= 3.5.1"
   s.add_dependency 'activesupport'
 
-  # Required for ActionController::DataStreaming::FileBody
-  s.add_dependency "actionpack", ">= 3.2"
+  # Required for ActionDispatch::Response::FileBody
+  s.add_dependency "actionpack", ">= 5.0"
```

Comment:
I'd like to continue supporting Rails 4.x, so we should revert this and find a different way to rely upon the correct class